jsdependency/jspackage.go

```diff
@@ -119,27 +123,27 @@ func toolCommandBuilder(packageManger Tool, command managerCommand) string {
 	return "add"
 }
 
-func createManagerCmd(packageManager Tool, packageManagerCmd string, commandScope CommandScope, pkg ...string) (*command.Model, error) {
+func createManagerCmd(packageManager Tool, packageManagerCmd string, commandScope CommandScope, pkg ...string) (command.Command, error) {
+	var commandName string
 	var commandArgs []string
 	switch packageManager {
 	case Npm:
-		commandArgs = []string{"npm", packageManagerCmd}
+		commandName = "npm"
+		commandArgs = []string{packageManagerCmd}
 		if commandScope == Global {
 			commandArgs = append(commandArgs, "-g")
 		}
 		commandArgs = append(commandArgs, pkg...)
 		commandArgs = append(commandArgs, "--force")
 	case Yarn:
-		commandArgs = []string{"yarn"}
+		commandName = "yarn"
+		commandArgs = []string{}
 		if commandScope == Global {
 			commandArgs = append(commandArgs, "global")
 		}
 		commandArgs = append(commandArgs, packageManagerCmd)
 		commandArgs = append(commandArgs, pkg...)
 	}
-	cmd, err := command.NewFromSlice(commandArgs)
-	if err != nil {
-		return nil, fmt.Errorf("Command creation failed, error: %s", err)
```

Comment:
Since this block is removed, the function no longer needs to have an error return type

Reply:
AddCommand and RemoveCommand exported functions also return an error, but to prevent breakage I would leave those as is for now.